encoder-otel-brave/README.md

```diff
@@ -0,0 +1,8 @@
+# encoder-otel-brave
+
+This encodes brave spans into OTLP proto format.
```

Comment:
copy this part into the PR desc for convenience

Reply:
Updated